### PR TITLE
json to jq

### PR DESCRIPTION
**Pull request by Kdisimone**
wait...let me do other file and then test

### CODE REVIEW

File: bin/oref0-autotune.sh [hunk 1]
```diff
@@ -153,7 +153,7 @@ else
     cp -up settings/pumpprofile.json settings/profile.json || die "Cannot copy settings/pumpprofile.json"
 fi
 # If a previous valid settings/autotune.json exists, use that; otherwise start from settings/profile.json
-cp settings/autotune.json autotune/profile.json && cat autotune/profile.json | json | grep -q start || cp autotune/profile.pump.json autotune/profile.json
+cp settings/autotune.json autotune/profile.json && cat autotune/profile.json | jq | grep -q start || cp autotune/profile.pump.json autotune/profile.json
```
Comment (scottleibrand) [inline bin/oref0-autotune.sh]: The syntax you want here is `jq .`

Comment (Kdisimone) [reply bin/oref0-autotune.sh]: isn't that the same as I proposed?  Did I miss something?  

I was about to look at the ns-get.sh next

Comment (scottleibrand) [reply bin/oref0-autotune.sh]: The `.` is important.

Comment (Kdisimone) [reply bin/oref0-autotune.sh]: doh,  saw that twice and twice i said "I wonder if that dot is meant to be left or right of the single backquote."  And twice I guessed wrong.  LOL

**Comment by Kdisimone** [comment]
How about the line 60 in ns-get.sh?  I'm not seeing a json issues there to fix?

**Comment by Kdisimone** [comment]
```
Kathryns-iMac:oref0 iMac4K$ oref0-autotune --dir=~/myopenaps --ns-host=https://testopenaps.herokuapp.com
Warning: API_SECRET is not set when calling oref0-autotune.sh
Running oref0-autotune --dir=/Users/iMac4K/myopenaps --ns-host=https://testopenaps.herokuapp.com --start-date=2017-10-05 --end-date=2017-10-05
Grabbing NIGHTSCOUT treatments.json and entries/sgv.json for date range...
Query: https://testopenaps.herokuapp.com find%5Bdate%5D%5B%24gte%5D=1507186800000&find%5Bdate%5D%5B%24lte%5D=1507273200000&count=1000
/usr/local/bin/ns-get: line 60: json: command not found
Couldn't download ns-entries.2017-10-05.json
```

**Comment by scottleibrand** [comment]
https://github.com/openaps/oref0/blob/master/bin/ns-get.sh#L60 calls https://github.com/openaps/oref0/blob/master/bin/ns-get.sh#L12, so it probably needs changed there.

**Comment by Kdisimone** [comment]
voila!

```
Parameter      | Pump     | Autotune 
-------------------------------------
ISF [mg/dL/U]  | 45.000   | 46.161   
Carb Ratio[g/U]| 7.000    | 7.280    
Basals [U/hr]  | -        |
  00:00        | 0.800    | 0.801    
  01:00        |          | 0.801    
  02:00        |          | 0.807    
  03:00        |          | 0.807    
  04:00        |          | 0.824    
  05:00        |          | 0.904    
  06:00        |          | 0.898    
  07:00        |          | 0.843    
  08:00        | 1.200    | 1.148    
  09:00        |          | 1.179    
  10:00        |          | 1.227    
  11:00        |          | 1.213    
  12:00        |          | 1.202    
  13:00        |          | 1.210    
  14:00        |          | 1.217    
  15:00        |          | 1.207    
  16:00        |          | 1.193    
  17:00        |          | 1.193    
  18:00        |          | 1.193    
  19:00        |          | 1.159    
  20:00        |          | 1.159    
  21:00        |          | 1.159    
  22:00        | 0.800    | 0.839    
  23:00        |          | 0.839    
Kathryns-iMac:oref0 iMac4K$ 
```

**Pull request merged**